extensions/amp-analytics/amp-analytics.md

```diff
@@ -685,6 +685,49 @@ Referrer policy is only available for `image` transport. If `referrerPolicy: no-
 }
 ```
 
+#### Linkers
+The `linkers` configuration object is used to enable cross domain ID syncing. `amp-analytics` will use this configuration to create an unique query parameter that can be used to join sessions. This parameter will then be appended to the specified outgoing links.
+
+```javascript
+"linkers": {
+  <paramName>: {
+    ids: <Object>,
+    proxyOnly: <boolean>,
+    destinationDomains: <Array<string>>,
+    enabled: <boolean>
+  }
+}
+```
+
+- `paramName` - This user defined name determines the name of the created query parameter.
+- `ids` - An object containing key-value pairs that will be passed along.
+- `proxyOnly` - (optional) Flag indicating whether the links should only be decorated on pages served on a proxy origin. Defaults to `true`.
+- `destinationDomains` - (optional) Links will be decorated if their domains are included in this array. Defaults to `canonical` and `source` domains.
+- `enabled` - Publishers must explicity set this to `true` to opt-in to using this feature.
+
+Example:
+```javascript
+"linkers": {
+  "linker1" : {
+    "ids" : {
+      "cid": "CLIENT_ID(_ga)",
+      "uid" "QUERY_PARAM(uid)",
+    }
+    "proxyOnly" : false,
+    "enabled": true,
+  }
+}
+```
+
+The above config would generate a parameter in this structure: `linker1=1~1clz2wh~cid~amp-SlB2gfii_11dWqtSr3zT4w~uid~1234`.
+
+There are several parts to this url. First you will see that the name of the param `linker1` matches the name from the configuration above. The param value contains several pieces of data seperated by the `~` delimiter.
+- The first value `1` indicates the version number of the linker used.
+- The second value `1clz2wh` is a hash generated by the linker.
```

Comment:
Should we go into how we generate hash here?

Reply:
"hash" -> "checksum"

and yes, we need to explain the checksum algorithm, and how pub can use it to prevent accidental ID sharing.

Reply:
Done.